loadsbroker/extensions.py

```diff
@@ -265,7 +271,8 @@ def debug(msg):
                        for x in volume_list if x and len(x) >= 2}
 
         def run(instance, tries=0):
-            dns = getattr(instance.state, "dns_server", [])
+            dns = getattr(instance.state, "dns_server", None)
```

Comment:
This is a fix for the docker API now strictly requiring dns to be a list (i think the older docker-py allowed strings). So I don't want e.g. [None], I want [] or [something]